src/main/java/bio/terra/service/resourcemanagement/azure/AzureResourceConfiguration.java

```diff
@@ -163,7 +164,7 @@ public record Synapse(
       int connectRetryInterval,
       int connectRetryCount) {}
 
-  public record Threading(int numTableThreads, int maxQueueSize) {}
+  public record Threading(int numTableThreads, int maxQueueSize, boolean blockWhenSaturated) {}
 
   @Bean("azureTableThreadpool")
```

Comment:
All of TDR's usages of this threadpool are within Stairway flight execution, so we favor blocking on saturation rather than failing fast.  If we were using a threadpool as part of servicing synchronous API responses, then we may favor the default rejection strategy.  I'll keep this in mind as I evaluate other existing threadpools.